free-programming-books-bl.md

```diff
@@ -12,6 +13,11 @@
 * [Computer Programming](http://cpbook.subeen.com/p/blog-page.html) - Tamim Shahriar Subeen 
 
 
+### Developer Cheatsheet
+
+* [ডেভসংকেত: বাংলা চিটশিটের ভান্ডার](https://devsonket.com/) - Devsonket Team
```

Comment:
Please remove the trailing slash from root URLs!

```suggestion
* [ডেভসংকেত: বাংলা চিটশিটের ভান্ডার](https://devsonket.com) - Devsonket Team
```

Reply:
Yes! Thank you, I have been looking so long for finding what's wrong!